operator/build/Dockerfile.ansible-s390x

```diff
@@ -0,0 +1,31 @@
+#FROM quay.io/operator-framework/ansible-operator:v0.11.0
+FROM quay.io/mew2057/ansible-operator:latest
+MAINTAINER jdunham@us.ibm.com
+
+LABEL name="IBM Spectrum Scale CSI Operator" \
+      vendor="ibm" \
+      version="1.0.1" \
+      release="1" \
+      run='docker run ibm-spectrum-scale-csi-operator' \
+      summary="An Ansible based operator to run and manage the deployment of the IBM Spectrum Scale CSI Driver." \
+      description="An Ansible based operator to run and manage the deployment of the IBM Spectrum Scale CSI Driver." 
+
+#
+#RUN REPOLIST=ubi-7,ubi-7-optional \
+#    yum -y update-minimal --disablerepo "*" --enablerepo ubi-7 --disableplugin=subscription-manager --setopt=tsflags=nodocs \
+#      --security --sec-severity=Important --sec-severity=Critical && \
+#    yum -y install --disablerepo "*" --enablerepo ${REPOLIST} --disableplugin=subscription-manager --setopt=tsflags=nodocs ${INSTALL_PKGS} && \
+#    yum clean all
+
+
+COPY build/health_check.sh .
+#RUN chmod 777 ./health_check.sh
```

Comment:
It looks like `health_check.sh` is set to 755 in the repo, so probably OK  to remove this

Reply:
Removed.